woocommerce/woocommerce-ios
# Help & Support: Add the app log feature

Closes #618.

This PR adds the ability to view, share, and send application logs. 

## To Test
1. Log in and navigate to My store > Settings > Help & Support > View Application Log.
2. Select any log available and view it.
3. Select the share button and send it, save it, or share it. Verify it arrived at the expected place.
4. When ready, select the "Clear old activity logs" button. Verify only the "Current" row and application log remains.

<img src="https://user-images.githubusercontent.com/1062444/51944148-e3065400-23e0-11e9-9112-ab0ffcc539b8.gif" width="300" />

Update release notes:


cc: @bummytime or @astralbodies first review wins 😀
